src/backend/joanie/tests/core/test_models_organization.py

```diff
@@ -430,3 +430,17 @@ def test_models_organization_contracts_signature_link_empty(self):
             str(context.exception),
             "No contract to sign for this organization.",
         )
+
+    def test_models_organization_fields_phone_number_formatted(self):
+        """The `phone_number` field should be formatted without spaces on save."""
+        organization1 = factories.OrganizationFactory()
+        organization2 = factories.OrganizationFactory()
+
+        organization1.contact_phone = "+33 1 23 45 67 89"
+        organization1.save()
+
+        organization2.contact_phone = "01 23 45 67 89"
+        organization2.save()
+
+        self.assertEqual(organization1.contact_phone, "+33123456789")
+        self.assertEqual(organization2.contact_phone, "0123456789")
```

Comment:
```suggestion
        organization1 = factories.OrganizationFactory(contact_phone="+33 1 23 45 67 89")
        organization1.save()
        self.assertEqual(organization1.contact_phone, "+33123456789")

        organization2 = factories.OrganizationFactory(contact_phone="01 23 45 67 89")
        organization2.save()
        self.assertEqual(organization2.contact_phone, "0123456789")
```
As the clean methods does more than just removing spaces, maybe we should add more tests ?

Reply:
I've added more tests combining different inputs of 'contact_phone' values.